react/components/EditorContainer/Sidebar/ConfigurationList/index.tsx

```diff
@@ -465,19 +466,21 @@ class ConfigurationList extends React.Component<Props, State> {
         iframeRuntime.extensions
       )
 
-      const newActiveExtension = clone({
-        ...partialNewActiveExtension,
-        content: getSchemaPropsOrContent({
-          i18nMapping,
-          isContent: true,
-          messages: iframeRuntime.messages,
-          properties: this.componentProperties,
-          propsOrContent: partialNewActiveExtension.content,
-          shouldTranslate: false,
-        }),
-      })
+      const newListContent = newQueryData.data.listContentWithSchema
+
+      const newConfigurations = (newListContent && newListContent.content) || []
 
-      this.activeExtension = newActiveExtension
+      const newActiveConfiguration =
+        newConfigurations.find(item =>
+          partialNewActiveExtension.contentMapId.startsWith(item.contentId)
```

Comment:
why is this `startsWith` instead of `===`?

Reply:
Because here's what they look like:

`contentMapId`:

```
s8Ah8D2Uu56sxSNJfg6mpJ
```

`contentId`:

```
s8Ah8D2Uu56sxSNJfg6mpJ~bGjcshQCaP391mvVdLJFBh
```